### PR TITLE
Create Dockerfile

### PR DESCRIPTION
**Pull request by JosephLaiC**
After struggling with build mace with bioconda or manually build with biocontainer, create mace dockerfile based on the centos7.

 # Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [ ] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [x] LABEL about.documentation   
  - [ ] LABEL about.license_file
  - [x] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata  

Other message:
After convert quay.io/biocontainers/mace to singularity sif, couldn't bind directory with error:
/etc/resolv.conf doesn't exist in container

while run "conda install -c bioconda mace" showed error:
- mace -> gsl >=2.5,<2.6.0a0 
- mace -> python_abi 2.7.\* \*\_cp27mu

I've checked this version of docker could work in singularity with volume binding.

### CODE REVIEW

File: mace/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+################## BASE IMAGE ######################
+
+FROM biocontainers/biocontainers:v1.0.0_cv4
+
+################## METADATA ######################
+
+LABEL base_image="biocontainers:v1.0.0_cv4"
+LABEL version="3"
```
Comment (osallou) [inline mace/Dockerfile]: First release of dockerfile so should be 1

**Review by osallou — CHANGES_REQUESTED** [review]
Should be in a subdir with upstream version name

**Comment by osallou** [comment]
> Should be in a subdir with upstream version name

you misunderstood me, must be in softwarename/softwareversion/Dockerfile

in your case:

mace/1.2/Dockerfile

and software.name should be mace (lowercase)

**Comment by JosephLaiC** [comment]
Just a new guy in github. lol

**Comment by osallou** [comment]
as suggested, license field should be spdx compliant ie 

GPL-3.0-or-later or GPL-3.0-only (depends on soft license)

and should see if biotools entry match this software to add related metadata label

**Comment by JosephLaiC** [comment]
> should see if biotools entry match this software to add related metadata label

MACE on biotools is "MACE - Mass Spectra for Chemical Ecology".

This software is "MACE - Model base Analysis of ChIP-Exo".

**Pull request merged**